.github/ISSUE_TEMPLATE/add_plugin.md

```diff
@@ -59,3 +53,15 @@ If your plugin isn't production ready, please leave a comment stating whether yo
 
 - [ ] The plugin must set a `min_server_version` in the manifest.
 - [ ] The plugin must work on all Mattermost versions greater than or equal to the `min_server_version`.
+
+**Documentation requirements (checked by a Technical Writer)**
+
+- [ ] The plugin must include detailed usage documentation with at least one screenshot of the plugin in action, list of features, and a development guide. This is typically a `README` file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [`README` of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md). Typical components of documentation include:
+
+    * Requirements/Prerequisites
+    * Installation steps
+    * Configuration steps
+    * Usage
+    * Troubleshooting
+    * Screenshots (if available)
+    * Link or email address for help/support
```

Comment:
Should the line from https://github.com/mattermost/mattermost-developer-documentation/pull/929/files#diff-01cbb01e3700f3a6f551852a98b49ab01ddff4600efe6642c6f813f803bfde3aR52 also get copied over?